src/core_plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -24,7 +24,7 @@ uiModules.get('apps/management')
     name: config.get('indexPattern:placeholder'),
     isTimeBased: true,
     nameIsPattern: false,
-    notExpandable: false,
+    expandable: false,
```

Comment:
To clarify, the idea here is to use `expandable` for new index patterns while upgrading-in-memory any previous usage of `notExpandable`?